CHANGELOG.md

```diff
@@ -2,6 +2,91 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.2.0
+
+**Release date:** 2023-12-05
+
+This minor release comes with API changes, bug fixes and several new features.
+
+### Bucket
+
+A new field, `.spec.prefix`, has been added to the Bucket API, which enables
+server-side filtering of files if the object's `.spec.provider` is set to
+`generic`/`aws`/`gcp`.
+
+### OCIRepository && HelmChart
```

Comment:
```suggestion
### OCIRepository and HelmChart
```

maybe? Not sure why `&` is picked over `&&`.

Reply:
sorry, was still in the coding headspace i guess